src/main/java/org/broadinstitute/hellbender/tools/walkers/mutect/FilteringFirstPass.java

```diff
@@ -0,0 +1,207 @@
+package org.broadinstitute.hellbender.tools.walkers.mutect;
+
+
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.VariantContext;
+import org.apache.commons.lang3.tuple.ImmutablePair;
+import org.apache.commons.lang3.tuple.Pair;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.tsv.DataLine;
+import org.broadinstitute.hellbender.utils.tsv.TableColumnCollection;
+import org.broadinstitute.hellbender.utils.tsv.TableWriter;
+import org.broadinstitute.hellbender.utils.variant.GATKVCFConstants;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
+
+public class FilteringFirstPass {
```

Comment:
Could you add some class javadoc?  Would like to know how this fits in the grand scheme of things...

Reply:
done